lib/manager/components/GeneralSettings.js

```diff
@@ -199,6 +201,9 @@ export default class GeneralSettings extends Component<Props, State> {
             </ListGroupItem>
           </ListGroup>
         </Panel>
+        <Panel header={<h3>Labels</h3>}>
+          <LabelAssigner feedSource={feedSource} project={project} />
+        </Panel>
```

Comment:
At first I was expecting this to be available within a separate tab pane within the ProjectSettings component. I think it makes more sense to add this in it's own tab pane that can be reached via clicking on a link container. Something like this:

<img width="966" alt="Screen Shot 2021-07-23 at 9 56 32 PM" src="https://user-images.githubusercontent.com/3112493/126857842-a044dc3b-e247-499a-a521-868c6bfe600c.png">

Reply:
Was basing the pr on [the mockups](https://www.sketch.com/s/8a229376-9d05-40ad-b1e6-04b4433b965f/a/pa885bp) but I do prefer what you have here. @landonreed what do you think?

Reply:
Given that we're showing the label manager on the dashboard/ProjectViewer, I don't think it's also needed on the Settings tab. We really only need to show it in one place (and I think that place should be next to the list of feed sources).

Reply:
Sounds good. Will remove it!

Reply:
Do you think it should be removed from the feed source settings as well?

Reply:
Oh, didn't even notice it was there. Yea, I think we should remove that panel also.